setup.json.example

```diff
@@ -1,5 +1,4 @@
 {
-    "wordpress_version": "4.4",
```

Comment:
I'd just comment it out here so it's obvious where it goes. (And maybe comment on its purpose.)